### PR TITLE
RDK-36782: Cpeabs changes for RDK-V devices - webconfig,json

### PR DESCRIPTION
**Pull request by BalaNaveenaNivetha**
[RDK-36782: Cpeabs changes for RDK-V devices - webconfig, json file]

### CODE REVIEW

**Review by karuna2git — APPROVED** [review]

File: src/rdkv/impl.c [hunk 1]
```diff
@@ -51,7 +51,7 @@
 #define PARAM_RFC_ENABLE "eRT.com.cisco.spvtg.ccsp.webpa.WebConfigRfcEnable"
 
 #define WEBCFG_PARTNER_JSON_FILE "/etc/partners_defaults_webcfg_video.json"
-#define WEBCFG_DB_STORE          "/opt/.webcfg.json"
+#define WEBCFG_DB_STORE          "/opt/.webconfig.json"
```
Comment (shilpa24balaji) [inline src/rdkv/impl.c:54]: Why is this change needed?

Comment (shilpa24balaji) [reply src/rdkv/impl.c:54]: Used to store the webconfig parameter values
